### PR TITLE
Update to SNI packaging

### LINKED ISSUE #12195: runtime.native.System.Data.SqlClient.sni in release/1.1.0 references RTM 1.0 packages
Currently the package depends on the following in 1.1:
```xml
      <dependency id="runtime.win10-arm64.runtime.native.System.Data.SqlClient.sni" version="4.3.0-beta-24514-00" />
      <dependency id="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni" version="4.0.1" />
      <dependency id="runtime.win7-x86.runtime.native.System.Data.SqlClient.sni" version="4.0.1" />
```

We need to understand if this is intentional or if folks were planning to build new versions of this component from the 1.1 codebase (presumably TFS?).

/cc @ianhays @saurabh500

### PR DESCRIPTION
**Pull request by saurabh500**
Updated the SNI packaging to reference the stable build so that 1.1 ships with latest SNI binaries

Fixes #12195

### CODE REVIEW

**Comment by saurabh500** [comment]
cc @weshaggard @gkhanna79 @dagood 
Please take a look.

File: src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj [hunk 1]
```diff
@@ -4,13 +4,14 @@
...
+    <PreReleaseLabel>servicing</PreReleaseLabel>
   </PropertyGroup>
   <ItemGroup>
     <Dependency Include="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni">
-      <Version>$(RuntimeWin7RuntimeNativeSystemDataSqlClientSniVersion)</Version>
+      <Version>4.4.0-beta-24612-00</Version>
     </Dependency>
     <Dependency Include="runtime.win7-x86.runtime.native.System.Data.SqlClient.sni">
-      <Version>$(RuntimeWin7RuntimeNativeSystemDataSqlClientSniVersion)</Version>
+      <Version>4.4.0-beta-24612-00</Version>
```
Comment (weshaggard) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: Yes lets try to update the RuntimeWin7RuntimeNativeSystemDataSqlClientSniVersion property instead.

File: src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj [hunk 2]
```diff
@@ -4,13 +4,14 @@
   <PropertyGroup>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
+    <PreReleaseLabel>servicing</PreReleaseLabel>
   </PropertyGroup>
   <ItemGroup>
     <Dependency Include="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni">
-      <Version>$(RuntimeWin7RuntimeNativeSystemDataSqlClientSniVersion)</Version>
+      <Version>4.4.0-beta-24612-00</Version>
```
Comment (weshaggard) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: Yes we will need to figure out how to mark it as stable once it is ready.

File: src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj [hunk 3]
```diff
@@ -4,13 +4,14 @@
   <PropertyGroup>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
+    <PreReleaseLabel>servicing</PreReleaseLabel>
```
Comment (weshaggard) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: This package shouldn't need to be marked with "servicing" as all the packages that build in the release build are already marked as "preview1" as they should (https://github.com/dotnet/corefx/blob/release/1.1.0/Packaging.props#L3).

The ones I thought we wanted to mark as servicing were the ones we are building in TFS. I'm not entirely sure we actually care to do that now that I"m looking at it. We should just keep them as "beta" as they are and then move them to stable once we are done with them.

Comment (gkhanna79) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: @weshaggard I thought this package contains sni.dll that is built in the closed build and thus, the package builds there and not in the open. Is that not correct?

My proposal is to mark this as servicing in the TFS branch (which corresponds to master of CoreCLR and not release/1.1.0) to indicate the nature of the package.

Comment (weshaggard) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: This package isn't the one that contains sni.dll itself, that is in the individual RID packages, this package just references all the RID split SNI packages.

Comment (gkhanna79) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: Ah, got it!

File: src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj (hunk 2 above)
Comment (saurabh500) [inline src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj]: @gkhanna79 Good catch.

**Comment by saurabh500** [comment]
@gkhanna79 @weshaggard @dagood I have updated the PR with the changes.

**Comment by gkhanna79** [comment]
LGTM.

**Pull request merged**